Ktisis/Overlay/Skeleton.cs

```diff
@@ -93,7 +94,7 @@ string Name
 					var boneColRgb = Ktisis.Configuration.GetCategoryColor(bone);
 					if (isUsing) boneColRgb.W = Math.Min(0.15f, boneColRgb.W);
 
-					var boneColor = ImGui.GetColorU32(boneColRgb);
+					var boneColor = ImGui.GetColorU32(boneColRgb.WithW(Ktisis.Configuration.SkeletonLineOpacity));
```

Comment:
It looks like this would overwrite the existing Alpha calculation if the Gizmo is in use above. Suggest moving the extension method call to `.GetCategoryColor` instead.

Reply:
> 

I didn't put it in this method, I just multipled the alpha by the opacity

Reply:
I meant putting it at the end of the `.GetCategoryColor` call, not *inside* the `GetCategoryColor` method.